docs/source/configuration/advanced_configuration.md

```diff
@@ -124,7 +124,7 @@ This section contains a set of guidance for advanced configuration requirements
 * [How to bypass the configuration loading rules](#how-to-bypass-the-configuration-loading-rules)
 * [How to use Jinja2 syntax in configuration](#how-to-use-jinja2-syntax-in-configuration)
 * [How to do templating with the `OmegaConfigLoader`](#how-to-do-templating-with-the-omegaconfigloader)
-* [How to use custom resolvers in the `OmegaConfigLoader`](#how-to-use-custom-resolvers-in-the-omegaconfigloader)
+* [How to use resolvers in the `OmegaConfigLoader`](#how-to-use-resolvers-in-the-omegaconfigloader)
```

Comment:
This query is also listed as an anchor in https://docs.kedro.org/en/stable/faq/faq.html -- please could you update the page there too (there will be a break otherwise)?